.github/workflows/test.yml

```diff
@@ -14,11 +14,12 @@ jobs:
     - name: Use Node.js
       uses: zendesk/setup-node@v2.1.2
       with:
-        node-version: '12.x'
+        node-version: '14.x'
     - run: yarn install
     - run: yarn lint
     - run: yarn test
     - run: yarn test:functional
+    - run: yarn link:bin && yarn run zcli
```

Comment:
Just `yarn dev` will do the same, linking is not needed (its usually only for creating symlink to map global zcli to local copy)